src/lib/diagnostics.ts

```diff
@@ -96,11 +96,11 @@ export class DiagnosticManager {
                 diagnostic.relatedInformation = [
                     new vscode.DiagnosticRelatedInformation(
                         new vscode.Location(vscode.Uri.parse(violation.url), range),
-                        `Current Code: ${apexGuruViolation.currentCode}`
+                        `\n// Current Code: \n ${apexGuruViolation.currentCode}`
```

Comment:
Did you mean to have a space after the second newline? `\n `? If so I'm assuming that is just for display purposes?

Reply:
My bad! Fixed this.